CHANGELOG.md

```diff
@@ -11,6 +11,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix switching to master when we're already on master
 - Add MacOS testing
 
+## 2.0.1 (2020-05-21)
+
+- fix broken environment property (needs a Hash, was set to String)
+- update documentation pages with new custom resource name ruby_build_definition
+- make the automatic addition of the Ruby version to the prefix an option
+- allow users to use the old ruby_build_ruby resource name, for compatibility
+- do not assume that users are running Chef 15.3.x and can use unified_mode
```

Comment:
didn't `unified_mode`, come in, in the last version of 14?

Reply:
15.3.0. Thing is, a lot of companies (including the one I work for) struggle to keep up with Chef's rapid-fire releases of the Chef client. I've got nodes still running 13.x. IMHO it's better to stay compatible if possible rather than just bugging out.

https://docs.chef.io/release_notes/#custom-resource-unified-mode

Reply:
(OTOH if unified_mode is more important here than I realize—if something will break if it's not there—I'm not married to the `#respond_to?` clause.)

Reply:
Switching it all to unified_mode without all the drudge, would be the best way IMO here. 

Our support policy is currently 15+, and in this case I’d strongly suggest that unified_mode makes this work so nicely, that we should just support whatever minimum version of 15 that supports it.